### DISCUSSION TITLE
Alternative APIs for Python example

### DISCUSSION THREAD

**Discussion by Toh-Bhydill** [modelcontextprotocol/.github#66] [Ideas]
### Pre-submission Checklist

- [X] I have verified this would not be more appropriate as a feature request in a specific repository
- [X] I have searched existing discussions to avoid duplicates

### Your Idea

Perhaps consider including alternative, non-commercial APIs for use in the demo examples?  I'm sure OpenWeather is a fine organization, however they ask for _a lot_ of information on account creation and have several terms-of-service to read through.

Neither of which something the [National Weather Service](https://www.weather.gov/documentation/services-web-api) API does.

### Scope

- [ ] Protocol Specification
- [ ] SDK Features
- [ ] Documentation
- [X] Developer Experience
- [X] Other

**Comment by jspahrsummers** [comment]
Thanks, agreed! We're addressing this in some doc updates happening now: https://github.com/modelcontextprotocol/docs/pull/63
